composer.d10.json

```diff
@@ -34,8 +36,10 @@
         "drupal/webform": "dev-6.2.x",
         "drush/drush": "^11",
         "oomphinc/composer-installers-extender": "^2.0",
+        "opis/json-schema": "^2.3",
         "vlucas/phpdotenv": "^5.1",
-        "webflo/drupal-finder": "^1.2"
+        "webflo/drupal-finder": "^1.2",
+        "wikimedia/composer-merge-plugin": "^2.0"
```

Comment:
Added `composer-merge-plugin` to D9 and D10 composer files and completed a `composer update --lock` to implement the merge.

Reply:
@richardgaunt 
the packages above are still not removed - can you please remove

Reply:
Done - building now on CI.